### PR TITLE
XW-4939 | 

### PR DESCRIPTION
**Pull request by rybmat**
## Links

* https://wikia-inc.atlassian.net/browse/XW-4939
* https://github.com/Wikia/app/pull/15389

## Description

Support for other file pages than image / video / ogg

## Reviewers

Use `@` mentions here. Alternatively, assign the pull request to a person.

### CODE REVIEW

File: app/styles/component/_article-media-thumbnail.scss [hunk 1]
```diff
@@ -33,6 +33,7 @@
 		}
 	}
 
+	&.downloadable figcaption,
 	&.is-ogg figcaption {
```
Comment (hakubo) [inline app/styles/component/_article-media-thumbnail.scss:37]: looks like this class has been removed?

File: mirage/fixtures/file-page.js [hunk 2]
```diff
@@ -24,10 +24,12 @@ export default {
 				type: 'image',
 				url: 'data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==',
 				fileUrl: 'http://muppet.igor.wikia-dev.pl/wiki/File:Kerm.jpg',
-				title: 'Kerm.jpg',
+				title: 'jb vjbvhv .jpg',
```
Comment (hakubo) [inline mirage/fixtures/file-page.js]: just a test data, I know
but maybe something nicer? :)

File: app/styles/component/_article-media-thumbnail.scss (hunk 1 above)
Comment (rybmat) [reply app/styles/component/_article-media-thumbnail.scss:37]: @hakubo it is used in article-content, comes from app

File: tests/acceptance/file-page-test.js [hunk 3]
```diff
@@ -21,14 +21,15 @@ module('Acceptance | file page', (hooks) => {
 	});
 
 	test('visiting File Page', async (assert) => {
+		await visit('/');
```
Comment (hakubo) [inline tests/acceptance/file-page-test.js:24]: did you open ticket to fix that?

Comment (rybmat) [reply tests/acceptance/file-page-test.js:24]: I'm in the middle

Comment (rybmat) [reply tests/acceptance/file-page-test.js:24]: https://wikia-inc.atlassian.net/browse/XF-565

**Review by JamesHawking — APPROVED** [review]

**Review by bkoval — APPROVED** [review]

File: app/styles/component/_article-media-thumbnail.scss (hunk 1 above)
Comment (hakubo) [reply app/styles/component/_article-media-thumbnail.scss:37]: !ok

**Pull request merged**